maxim/tests/test_test_runs.py

```
@@ -241,5 +263,597 @@ def error(self, message: str, e: Optional[Exception] = None) -> None:
         if result is not None:
             self.assertListEqual(result.failed_entry_indices, [])
 
+    def test_create_test_run_with_image_variables_and_prompt_workflow(self):
+        """Test prompt workflow and chains with image variable columns and other variable types"""
+
+        # Test data with multiple variable types including image URLs
+        data: Data = [
```

Pylint (3.3.7)

[warning] Redefining name 'data' from outer scope (line 24)

(W0621)